SIT.Manager/Views/MainView.axaml

```diff
@@ -51,98 +66,70 @@
 				</ItemsControl.ItemTemplate>
 			</ItemsControl>
 		</StackPanel>
-		
-		
 
-		<ui:NavigationView PaneDisplayMode="Left"
-						   IsBackButtonVisible="False"
-						   IsBackEnabled="False"
-						   IsSettingsVisible="False"
-						   IsPaneToggleButtonVisible="False"
-						   OpenPaneLength="166"
-						   Grid.Row="1"
-						   Name="NavView"
-						   ItemInvoked="NavView_ItemInvoked">
-			<ui:NavigationView.MenuItems>
-				<ui:NavigationViewItem IconSource="Play" 
-									   Content="{DynamicResource PlayTitle}" 
-									   IsEnabled="{Binding !IsInstallRunning}"
-									   Tag="SIT.Manager.Views.PlayPage" 
-									   ToolTip.Tip="{DynamicResource PlayTitleToolTip}"/>
+		<SplitView Background="Black"
+				   IsPaneOpen="True"
+				   DisplayMode="Inline"
+				   OpenPaneLength="166"
+				   Grid.Row="1">
+			<SplitView.Pane>
+				<Grid RowDefinitions="Auto, *, Auto"
+					  Background="Black">
```

Comment:
I tried out `#080808` and IMO it looks a little nicer. Could give it a try and see how ya feel about it? :3

Reply:
okie I went with it.